lib/Config.js

```diff
@@ -435,7 +436,11 @@ class Config {
             if (process.env.S3AUTH_CONFIG) {
                 authfile = process.env.S3AUTH_CONFIG;
             }
-            const authData = require(authfile);
+            let authData = require(authfile);
+            if (process.env.ACCESS_KEY && process.env.SECRET_KEY) {
```

Comment:
Suggestion, why not use `SCALITY_ACCESS_KEY_ID` and `SCALITY_SECRET_ACCESS_KEY` which would be similar to `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` used by AWS. When we have multiple backends we can use both sets to work with AWS/SCALITY.